### PR TITLE
Add CLI build

### PR DESCRIPTION
**Pull request by mimartin12**
## Type of change

- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [X] Build/deploy pipeline (DevOps)
- [ ] Other

## Objective
Monorepo support for CLI workflows.

## Code changes

- **.github/workflows/build-cli.yml:** Updated workflow for monorepo support, updated workflow to handle new `pkg`version. Updated actions and applied linting.
- **.github/workflows/release-browser.yml:** Updates browser release name.
- **.github/workflows/release-cli.yml:** Updated workflow for monorepo support, updated release name as well.
- **apps/cli/.github/*:** Removed uneeded folder.
## Before you submit

- [ ] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] I have added **unit tests** where it makes sense to do so (encouraged but not required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)

### CODE REVIEW

File: .github/workflows/build-cli.yml [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+
+  cli:
+    name: Build CLI
+    runs-on: windows-2019
+    needs:
+      - setup
+    env:
+      _PACKAGE_VERSION: ${{ needs.setup.outputs.package_version }}
+      _WIN_PKG_FETCH_VERSION: 16.14.2
+      _WIN_PKG_VERSION: 3.3
```
Comment (Hinton) [inline .github/workflows/build-cli.yml:68]: We accidentally bumped pkg to 5.7 meaning we also need to change these.

File: .github/workflows/release-cli.yml [hunk 2]
```diff
@@ -25,15 +29,15 @@ jobs:
       - name: Branch check
         if: ${{ github.event.inputs.release_type != 'Dry Run' }}
         run: |
-          if [[ "$GITHUB_REF" != "refs/heads/rc" ]] && [[ "$GITHUB_REF" != "refs/heads/hotfix-rc" ]]; then
+          if [[ "$GITHUB_REF" != "refs/heads/rc" ]] && [[ "$GITHUB_REF" != "refs/heads/hotfix-rc/*" ]]; then
```
Comment (vgrassia) [inline .github/workflows/release-cli.yml:32]:
```suggestion
          if [[ "$GITHUB_REF" != "refs/heads/rc" ]] && [[ $GITHUB_REF != refs/heads/hotfix-rc/* ]]; then
```

The second comparison does not work properly with quotes.

File: .github/workflows/release-cli.yml [hunk 3]
```diff
@@ -66,28 +72,28 @@ jobs:
...
+                      apps/cli/bw-linux-${{ env.PKG_VERSION }}.zip,
+                      apps/cli/bw-linux-sha256-${{ env.PKG_VERSION }}.txt,
+                      apps/cli/bitwarden-cli.${{ env.PKG_VERSION }}.nupkg,
+                      apps/cli/bw_${{ env.PKG_VERSION }}_amd64.snap,
+                      apps/cli/bw-snap-sha256-${{ env.PKG_VERSION }}.txt"
           commit: ${{ github.sha }}
-          tag: v${{ env.PKG_VERSION }}
-          name: Version ${{ env.PKG_VERSION }}
+          tag: cli-v${{ env.PKG_VERSION }}
+          name: CLI ${{ env.PKG_VERSION }}
```
Comment (vgrassia) [inline .github/workflows/release-cli.yml:96]:
```suggestion
          name: CLI v${{ env.PKG_VERSION }}
```

Comment (vgrassia) [reply .github/workflows/release-cli.yml:96]: We might need this in the `browser` and `desktop` Release workflows as well.

**Review by vgrassia — APPROVED** [review]

**Pull request merged**